File: reto4/es_primo.c
#include<stdio.h>

int main() 
{
    int myNum, control = 0, aux;
    printf("Write a numbre. Is a prime number?: \n");
    scanf("%d", &myNum);

    for ( int i = 2; i < myNum && control!= 1; i++)
    {
        aux = myNum % i;
        if (aux == 0){
            control = 1;
        }
    }

    if (control == 1){
        printf("%i No es primo", myNum);
    }
    else {
        printf("%i Si es primo", myNum);
    }
    return 0;
}

File: reto4/Esprimo.java
import java.util.Scanner;
public class Esprimo{

    public static void main(String[] args){
        Integer numberToEvaluate;
        boolean control = false;
        Scanner scan = new Scanner(System.in);
        System.out.println("write a number, Is this a prime number?: ");
        numberToEvaluate = scan.nextInt();

        if (numberToEvaluate == 0 || numberToEvaluate == 1){
            System.out.println("Is no a prime");
        }else if (numberToEvaluate == 2){
            System.out.print("Is a prime");
        }
        else{for (int i = 2; i < numberToEvaluate; i++) {
                if (numberToEvaluate % i == 0){
                    control = true;
                    }
                }
            if (control == false){
                System.out.println("Is prime");
            }else{
                System.out.println("Is not a prime");
            }
        }

    }
}